src/lib/parse/forbiddenChars.cpp

```diff
@@ -61,3 +61,36 @@ bool forbiddenChars(const char* s)
 
   return false;
 }
+
+
+
+/* ****************************************************************************
+*
+* forbiddenCharsInUriParam - 
+*/
+bool forbiddenCharsInUriParam(const char* s)
```

Comment:
It could pass a long time until we implement #1255... So, it is better to have this fixed, to avoid "misleading" names in functions in the code (especially for developers that no have been involved in this PR and who know this discussion :).

If we don't have better alternative than forbiddenCharsInUriParamWithoutSemicolonAndEqualSign(), let's use that.